packages/config/src/layer2s/loopring.ts

```diff
@@ -314,11 +314,11 @@ export const loopring: Layer2 = {
   },
   milestones: [
     {
-      name: 'Loopring Protocol 3.0 has launched',
+      name: 'First zkRollup is live',
```

Comment:
This milestone is in the scope of your project, you can just state what has been deployed